release/community-stable/clearlinux/meta.json

```diff
@@ -2,7 +2,7 @@
     "IsLinux" : true,
     "PackageFormat": "powershell-${PS_VERSION}-linux-x64.tar.gz",
     "SkipGssNtlmSspTests": true,
-    "SkipWebCmdletTests": false,
+    "IsBroken": true,
```

Comment:
```suggestion
    "SkipWebCmdletTests": false,
    "IsBroken": true,
```

Reply:
The default is false so this isn't needed